### ISSUE TITLE
.attrs feature not present

### ISSUE THREAD

**Issue by milesbeyond** [styled-components/vue-styled-components#68]
Code:
```
const Button = styled.button.attrs({
  title: 'TEST'
})`
  background: green
`;
```

Error:
```
Uncaught TypeError: vue_styled_components__WEBPACK_IMPORTED_MODULE_2__.default.button.attrs is not a function
```

I'm aware that this Vue port of Styled-Components is not fully featured, but would like to understand where/if this stands on the feature roadmap? It's a pretty killer feature IMO, if not *the* killer feature, and I miss it :(

Can someone shed some light onto this, please?

**Comment by liqueflies** [comment]
Hello,
thank you for the interest on the repo and sorry for late reply but I am very busy at the moment and no one is helping me on that.

Please, remember that vue-styled-components is a proof of concept, because it not depends on the main repo, but code is cloned and each new feature have to be re-implemented, so it's not an adapter of the main repo, obviously for a main repo limitation.

you can read this discussion starting from [here](https://github.com/styled-components/styled-components/issues/588#issuecomment-287300504) to understand the pain.

If this issue on main-repo still remain it's very hard to get updatd with all the new feature.

Also main repo is well maintained by a lot of developers, here I'm alone and I wold appreciate new maintainers that will help me keeping update this. :)

At the moment is very difficult to me get into new feature, I can review other PR's for clean code integration :)

Thank you so much,
Lorenzo

**Comment by m-sebastiyan** [comment]
Created a PR #83

**Comment by wellitongervickas** [comment]
https://github.com/wellitongervickas/batmovies/commit/73d5efe706c5747e49a13ebb1119b8d3c470c803
https://github.com/wellitongervickas/batmovies/commit/73d5efe706c5747e49a13ebb1119b8d3c470c803#diff-d6f575eb4fa640ebe7a8c06e74ef47b5

I just import in others layouts/*.vue to reuse my global styles

**Comment by liqueflies** [comment]
Solved merging  #111
